composer.json

```diff
@@ -54,7 +56,8 @@
         "cs-check": "phpcs",
         "cs-fix": "phpcbf",
         "test": "phpunit --colors=always",
-        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml",
+        "static-analysis": "psalm --config=.psalm.xml.dist --shepherd --stats"
```

Comment:
There is no need to add `--config` to the script as it will automatically use the projects `.dist` if exists.

Reply:
I see. I've renamed `.psalm.xml.dist` -> `psalm.xml.dist` (removed leading dot).
Will push it soon